### PR TITLE
breaking test - constructor returning multiple instances

### PR DESCRIPTION
**Pull request by tinovyatkin**
After realising  hacky nature of Choices constructor (#682)  I've decided to try it a little this way and created this test, that currently is breaking. @jshjohnson  Is there something wrong in the test or in Choices?

### CODE REVIEW

File: src/scripts/choices.test.js [hunk 1]
```diff
@@ -15,6 +15,22 @@ describe('choices', () => {
...
+      document.body.innerHTML = `
+      <input data-choice type="text" id="inp1" />
+      <input data-choice type="text" id="inp2" />
+      <input data-choice type="text" id="inp3" />
+      `;
+
+      const inputs = document.querySelectorAll('[data-choice]');
+      expect(inputs.length).to.equal(3);
+
+      const choices = new Choices();
```
Comment (jshjohnson) [inline src/scripts/choices.test.js]:
```suggestion
      const choices = new Choices(inputs);
``` 
:)

Comment (tinovyatkin) [reply src/scripts/choices.test.js]: The whole point here is to construct from a string. If a pass nothing to Choices constructor then defaults is a string ‘[data-choice]’

Comment (tinovyatkin) [reply src/scripts/choices.test.js]: By the way, `new Choices(inputs)` will not work - Choices is not supporting directly `NodesList` as constructor parameter.

Comment (tinovyatkin) [reply src/scripts/choices.test.js]: The thing is that if the constructor never worked in the expected way with selector returning several elements, then you can adjust/fix its behaviour in a minor version as it cannot be considered as breaking change.

**Comment by jshjohnson** [comment]
@tinovyatkin see https://github.com/jshjohnson/Choices/pull/693 👍

**Pull request closed without merging**